tests/unit/src/main/scala/scala/meta/internal/metals/debug/Debugger.scala

```diff
@@ -47,6 +64,88 @@ final class Debugger(server: RemoteServer)(implicit ec: ExecutionContext) {
     server.disconnect(args).asScala.ignoreValue
   }
 
+  def setBreakpoints(
+      source: Source,
+      breakpoints: Array[SourceBreakpoint]
+  ): Future[SetBreakpointsResponse] = {
+    val args = new SetBreakpointsArguments
+    args.setSource(source)
+    args.setBreakpoints(breakpoints)
+    server.setBreakpoints(args).asScala
+  }
+
+  def step(threadId: Long, nextStep: DebugStep): Future[Unit] = {
+    nextStep match {
+      case DebugStep.Continue =>
+        val args = new ContinueArguments()
+        args.setThreadId(threadId)
+        server.continue_(args).asScala.ignoreValue
+      case DebugStep.StepIn =>
+        val args = new StepInArguments()
+        args.setThreadId(threadId)
+        server.stepIn(args).asScala.ignoreValue
+      case DebugStep.StepOut =>
+        val args = new StepOutArguments()
+        args.setThreadId(threadId)
+        server.stepOut(args).asScala.ignoreValue
+      case DebugStep.StepOver =>
+        val args = new NextArguments()
+        args.setThreadId(threadId)
+        server.next(args).asScala.ignoreValue
+      case cause =>
+        val error = s"Unsupported debug step $cause"
+        Future.failed(new IllegalStateException(error))
+    }
+  }
+
+  def stackFrame(threadId: lang.Long): Future[StackFrame] = {
```

Comment:
nit: let's use `import java.lang{Long => JLong}` instead

Reply:
Good catch. In fact, we should use the Scala `Long` here.